google/cloud/storage/internal/google_application_default_credentials_file.cc

```diff
@@ -51,11 +51,12 @@ std::string GoogleApplicationDefaultCredentialsFile() {
   }
   // There are probably more efficient ways to do this, but meh, the strings
   // are typically short, and this does not happen that often.
-  auto root = std::getenv(CREDENTIALS_HOME_VAR);
+  auto root = std::getenv(GoogleApplicationDefaultCredentialsHomeVariable());
```

Comment:
The CL description said you changed the code to make a copy of the char*  returned by getenv but I don't see a copy being made here?

Reply:
The `previous_` member variable is now a `std::string` (was a `char const*`), so assigning to it makes a copy, right?

Reply:
Thanks, I missed that.